File: app/src/main/java/com/starcut/starflightclient/UnregistrationResponse.java
package com.starcut.starflightclient;

public class UnregistrationResponse implements StarFlightResponse {
    private final Result result;

    UnregistrationResponse(Result result)
    {
        this.result = result;
    }

    public enum Result
    {
        /**
         * Unregistration was successful
         */
        OK,
        /**
         * The device was not registered in the first place.
         */
        NOT_REGISTERED;
    }
}

File: app/src/main/java/com/starcut/starflightclient/starflight/callbacks/UnregistrationResponse.kt
package com.starcut.starflightclient.starflight.callbacks

class UnregistrationResponse(private val result: Result) : StarFlightResponse {

    enum class Result {
        /**
         * Unregistration was successful
         */
        OK,
        /**
         * The device was not registered in the first place.
         */
        NOT_REGISTERED
    }
}
